src/Templates/cs/Templates/Templates/CSTypeEditor/CustomControlPackage/CustomControlPackage.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <RepositoryUrl>https://github.com/dotnet/winforms-designer</RepositoryUrl>
```

Comment:
This will be removed, right?

Reply:
Yes, we need to put in the link to the new one.